handler/create_user.go

```diff
@@ -0,0 +1,65 @@
+package handler
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"os"
+
+	"github.com/jmoiron/sqlx"
+	_ "github.com/lib/pq"
+)
+
+func CreateUser(w http.ResponseWriter, r *http.Request) {
+	db, err := sqlx.Connect("postgres", os.Getenv("POSTGRES_DSN"))
+	if err != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+		w.Write([]byte(fmt.Sprintf("db connection error: %s", err.Error())))
+		return
+	}
+
+	err = r.ParseForm()
+	if err != nil {
+		w.WriteHeader(http.StatusBadRequest)
+		w.Write([]byte(fmt.Sprintf("parse form error: %s", err.Error())))
```

Comment:
are you sure you want to return the error to the client? usually you log the error and return a controlled error message as to not leak your implementation to malicious requests

Reply:
please ensure you consider the same for other non-ok returns